runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -1316,6 +1316,10 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 			}
 			continue;
 		}
+		if (try_scan(&scan_start, "pretouchHeapOnExpand")) {
```

Comment:
actually, we want to match similar (if not identical) option from OpenJDK:  -XX:+AlwaysPreTouch

parsing of such (-XX:+/- style) a GC option is in gcParseXXArguments() in mmparse.cpp

Reply:
Should I rename "pretouchHeapOnExpand" to "AlwaysPreTouch" ? Or just follow the  (-XX:+/- style) should be fine. I recall there was a discussion on if we use the name "AlwaysPreTouch" we should provide feature match to "AlwaysPreTouch".

Reply:
Internal C flag is ok to stay pretouchHeapOnExpand. Just need to make the command line option change.

We should probably document it and specifically state that we touch only heap memory (and not Meta structures). It's not clear to me if OpenJDK does more than just heap or not.

Not sure what you mean by 'provide feature match'.